### PR TITLE
[tests-only] Remove passing tests from pending state (2)

### PR DESCRIPTION
**Pull request by saw-jan**
Remove passing tests out of pending state

### Related issue
Part of https://github.com/owncloud/owncloud-sdk/issues/1089

### CODE REVIEW

File: tests/signedUrlIntegrationTest.js [hunk 1]
```diff
@@ -28,15 +30,13 @@ describe('Signed urls', function () {
...
-          'Content-Type': 'application/xml; charset=utf-8'
-        },
+        headers: xmlResponseHeaders,
         body: new XmlBuilder('1.0', '', 'ocs').build(ocs => {
           ocs.appendElement('meta', '', (meta) => {
             return ocsMeta(meta, 'ok', '100', 'OK')
           })
             .appendElement('data', '', (data) => {
-              data.appendElement('user', '', MatchersV3.equal(username))
+              data.appendElement('user', '', MatchersV3.string(username))
```
Comment (saw-jan) [inline tests/signedUrlIntegrationTest.js:39]: in oCIS, `user` will have userId value not a username

File: tests/helpers/pactHelper.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ const shareResponseOcsData = function (node, shareType, id, permissions, fileTar
     .appendElement('uid_file_owner', '', MatchersV3.string(config.adminUsername))
     .appendElement('displayname_file_owner', '', MatchersV3.string(config.adminUsername))
     .appendElement('path', '', MatchersV3.equal(fileTarget))
-    .appendElement('file_target', '', MatchersV3.equal(fileTarget))
+    .appendElement('file_target', '', MatchersV3.regex(`(\\/Shares)?\\${fileTarget}`, fileTarget))
```
Comment (individual-it) [inline tests/helpers/pactHelper.js:58]: the issues with that is, if the fileTarget has special char that have some meaning in regex it does not work 
e.g.
`/test space and + and #.txt` does not match `(\/Shares)?\/test space and + and #.txt`
for that the regex should be `(\/Shares)?\/test space and \+ and #\.txt`

Comment (individual-it) [reply tests/helpers/pactHelper.js:58]: so would need to adjust the regex or maybe `MatchersV3.includes` will be enough
includes | value: string | Value that must include the example value as a substring.

Comment (saw-jan) [reply tests/helpers/pactHelper.js:58]: Ohh, I was wondering why all the changed passed locally but not in CI.
It looks like I have not touched the test that uses this interaction.

**Review by individual-it — APPROVED** [review]

**Pull request merged**